locations/templates/locations/create.html

```diff
@@ -15,4 +33,7 @@
 
 {% block extra_media %}
     {{ form.media }}
+    <script>
+        activate_dynamic_inline_forms("racedistribution");
```

Comment:
This should be wrapped in `$(document).ready();` to avoid errors with resource loading and presentation
```suggestion
        $(document).ready(function() {
            activate_dynamic_inline_forms("racedistribution");
        });
```